SDK v3 Examples/ios-sdk-app/ios-sdk-app/ViewController+MPIMapViewDelegate.swift

```diff
@@ -87,16 +87,54 @@ extension ViewController: MPIMapViewDelegate {
 //        updateBlueDotBanner(blueDot: blueDot)
     }
 
-    func onDataLoaded(data: MPIData) {}
+    func onDataLoaded(data: MPIData) {
+        let rankings: MPIRankings? = data.rankings
+        let polygonRank = data.polygons[0].rank
+    }
 
     func onFirstMapLoaded() {
         self.onMapLoaded()
 
+        // get default camera state
         defaultRotation = mapView?.cameraControlsManager.rotation
         defaultTilt = mapView?.cameraControlsManager.tilt
 
+        // set camera state
         mapView?.cameraControlsManager.setRotation(rotation: 180)
         mapView?.cameraControlsManager.setTilt(tilt: 0)
+
+        // label all locations to be light on dark
+        mapView?.labelAllLocations(
+            options: MPIOptions.LabelAllLocations(
+                appearance: MPIOptions.LabelAppearance.lightOnDark
+            )
+        )
+
+        // create a multi-destination journey between 4 sample locations
+        guard let locations = mapView?.venueData?.locations else { return }
+        mapView?.getDirections(
+            to: MPIDestinationSet(destinations: [locations[4], locations[5], locations[6]]),
```

Comment:
This seems a little risky, if someone tries to point this at a venue that doesn't have 8 locations yet. Maybe do something like check if there are at least four locations and then make a shuffled copy of the locations array and use the first four

Reply:
I just thought they would replace the array accesses with more meaningful code before they tried it, but yeah makes sense. I simply added a check in the `guard` statement.

Reply:
Anything you put into sample code you can pretty much guarantee will end up in production at some point.  Go to https://thedubaimall.com/en/map, open the dev console, and click around. That's straight out of the web sdk sample app